request_builder.rb

```diff
@@ -5,20 +5,20 @@
 require_relative 'rosette_api_error'
 
 class RequestBuilder
-  attr_reader :user_key, :alternate_url, :params
+  attr_reader :alternate_url, :params, :user_key
 
-  def initialize(user_key, alternate_url, params={})
+  def initialize(user_key, alternate_url, params = {})
     @user_key = user_key
     @alternate_url = alternate_url
     @params = params
   end
 
   def prepare_plain_request(params)
-    uri = URI.parse(@alternate_url)
-    http = Net::HTTP.new(uri.host, uri.port)
-    http.use_ssl = true if uri.scheme == 'https'
+    uri = URI.parse @alternate_url
```

Comment:
In general if you can remove parentheses without removing clarity of the code and without introducing parsing errors, do so.